lib/oli_web/controllers/cognito_controller.ex

```diff
@@ -0,0 +1,143 @@
+defmodule OliWeb.CognitoController do
+  use OliWeb, :controller
+
+  import Oli.{HTTP, Utils}
+
+  require Logger
```

Comment:
We are requiring but not using this.

Reply:
Awesome will apply to next ticket! Great catch!